contracts/Gateway.sol

```diff
@@ -66,6 +67,11 @@ contract Gateway is ProtocolVersioned, Owned {
     uint256 public bounty;
     /** Storing utility token UUID */
     bytes32 public uuid;
+    /** Token conversion rate and decimal.
```

Comment:
Multi line comments should start below the line with the two asterisks. Please see [here](https://github.com/OpenSTFoundation/openst-protocol/blob/develop/SOLIDITY_STYLE_GUIDE.md#multi-line) for examples.